lib/apivore.rb

```diff
@@ -2,6 +2,7 @@
 require 'apivore/rspec_helpers'
 require 'apivore/swagger_checker'
 require 'apivore/swagger'
+require 'rspec/rails'
```

Comment:
I think ` require 'rspec'` should work if  `rspec/rails` is being used. Might be worth checking